aws-lambda-layer/Dockerfile

```diff
@@ -1,14 +1,19 @@
-FROM lambci/lambda:build-ruby2.5
+FROM amazonlinux:latest
 
 ARG FREETDS_VERSION=1.1.6
 
 ENV INSTALLDIR='/tmp/freetds'
+ENV TASKDIR='/var/task'
 ENV FREETDS_VERSION=$FREETDS_VERSION
 
 RUN yum update -y
 RUN yum install wget build-essential libc6-dev -y
+RUN yum install tar -y
+RUN yum install gzip -y
+RUN yum group install "Development Tools" -y
```

Comment:
Can these all be added to the existing "yum install" line? Doing so will keep all the dependencies in a single docker layer, which is preferable in this case. I'm guessing these tools were included in the other base image, but not in AL2, but we need them to build freetds?

Reply:
yes, these tools were included in the other base image, we need the C compiler, tar and gzip.